src/Http/CurlDispatcher.php

```diff
@@ -102,6 +103,8 @@ private function __construct(array $settings, RequestInterface $request)
             CURLOPT_COOKIEFILE => $cookies,
             CURLOPT_HEADERFUNCTION => [$this, 'writeHeader'],
             CURLOPT_WRITEFUNCTION => [$this, 'writeBody'],
+            CURLOPT_NOPROGRESS => false,
```

Comment:
Why add this option? According to php documentation, this should only be changed for debugging purposes only.

Reply:
This is necessary for the progress function in the following line, otherwise it is not called.

Reply:
I see.
In Embed 3.x I used the `write` callback for that (https://github.com/oscarotero/Embed/blob/v3.x/src/Http/CurlResult.php#L131) I think we can do the same here

Reply:
Great, that works. Now the response body is also empty.